Tests/L0/SonarQubePostTest/PRCA/PostCommentsTests.ps1

```diff
@@ -237,6 +251,15 @@ namespace Microsoft.TeamFoundation.SourceControl.WebApi
         public string LastMergeTargetCommit { get; set; }
     }        
 }
+
+namespace Microsoft.VisualStudio.Services.CodeReview.WebApi
+{
+    public class IterationChanges
+    {
+        
```

Comment:
Yes, I decided not to mock the logic related to code flow as it's a very low level API. This is enough for the test to pass.